core/roboconf-dm/pom.xml

```diff
@@ -81,8 +81,22 @@
 			<artifactId>junit</artifactId>
 			<scope>test</scope>
 		</dependency>
+
+		<dependency>
```

Comment:
The Messaging API bundle now have an adherence to the OSGi framework API. This is because of the `ReconfigurableClient` must be able to auto-connect to the `MessagingClientFactory` registry service. In "offline" mode (i.e. without an OSGi container), the dependency is still needed, but unused.